testmodule/src/simple.rs

```diff
@@ -0,0 +1,12 @@
+extern crate codechain_basesandbox as cbasesandbox;
+use cbasesandbox::execution::executee;
+use cbasesandbox::ipc::Ipc;
+
+#[cfg(all(unix, target_arch = "x86_64"))]
+fn main() -> Result<(), String> {
+    let mut ctx = executee::start::<cbasesandbox::execution::IpcUnixDomainSocket>();
+    let r = ctx.ipc.recv();
+    println!("CLIENT recv: {:?}", String::from_utf8(r));
+    ctx.ipc.send(b"I'm here!\0");
+    Ok(())
+}
```

Comment:
Reviewing what this code is doing seems not my role. Please request a review to others. Maybe seulgi?